modules/acquia_cms_common/composer.json

```diff
@@ -13,7 +13,7 @@
         "drupal/autologout": "^1.4",
         "drupal/config_ignore": "^3.0@beta",
         "drupal/config_rewrite": "^1.5",
-        "drupal/core": "^10.0.9",
+        "drupal/core": "^10.1",
```

Comment:
@rajeshreeputra Not sure about this change I guess "^10" would work.

Reply:
As discussed in DSM, minimum requirement is need to add as ^10.1